manifest.json

```diff
@@ -33,5 +33,8 @@
     "quick-find-search-links": {
       "description": "Search all links"
     }
-  }
+  },
+  "permissions": [
```

Comment:
Actually, I just had an a-ha moment after re-reading the docs. I had assumed that the `"tabs"` permission was required because I need to "populate the url […] propert[y] of `Tab`". However, `chrome.tabs.create()` does not actually take a `Tab`, it takes a `createProperties` that just happens to look a bit like the former. Same for `chrome.windows.create()`. I do access `sender.tab`, but no protected properties.